lib/user.js

```diff
@@ -29,7 +31,7 @@ const ensureUser = async function (claims) {
         firstName: result.firstName?.value,
         familyName: result.familyName?.value
       }, claims);
-      return { personUri: personUri };
+      return { personUri: personUri, personStatus: result.status.value };
     } else {
       const person = await insertNewUser(claims);
```

Comment:
Can we make `insertNewUser` return the status as well, so return value is in line with the case where the user already exists?